src/core/graph.js

```diff
@@ -1162,7 +1162,8 @@ export class Graph {
  */
 export function nodeToString(node: Node): string {
   const address = NodeAddress.toString(node.address);
-  return `{address: ${address}}`;
+  const timestamp = node.timestampMs || "null";
```

Comment:
In the case where the timestampMs is 0, this would improperly report null. `0` is a valid timestamp.

Recommend using `NullUtil.orElse`, or `node.timestampMs == null ? "null" : String(node.timestampMs)`

Reply:
accepted